### PR TITLE
[FR] Add "Ranking Queue", update "Beatmap ranking procedure"

### PR DESCRIPTION
**Pull request by DoryanR**
Proposal for the French translation of the wiki page `Ranking queue`.

Proposal to update the French translation of the wiki page `Beatmap ranking procedure`.

### CODE REVIEW

File: wiki/Beatmap_ranking_procedure/fr.md [hunk 1]
```diff
@@ -1,52 +1,43 @@
...
 
-Avant d'être nominés pour le classement, les beatmaps doivent recevoir [hype](/wiki/Glossary#hype) des messages d'au moins cinq utilisateurs différents.
+Avant d'être proposés au classement, les beatmaps doivent recevoir des messages de [hype](/wiki/Glossary#hype) d'au moins cinq utilisateurs différents.
 
 ## Nominations
 
-Les membres des [Beatmap Nominators](/wiki/People/The_Team/Beatmap_Nominators) (*BNs*) sont des modders expérimentés qui sont capables de nommer des beatmaps pour le classement. Leur rôle est de s'assurer qu'une beatmap répond aux critères du [Ranking Criteria](/wiki/Ranking_Criteria) ainsi que leurs propres normes de qualité personnelles.
+Les membres du groupe des [Beatmap Nominators](/wiki/People/The_Team/Beatmap_Nominators) (*BNs*) sont des moddeurs expérimentés capables de proposer des beatmaps classable. Leur rôle est de s'assurer qu'une beatmapset répond aux [critères de classement](/wiki/Ranking_Criteria) ainsi qu'à leurs propres critères de qualité.
```
Comment (TicClick) [inline wiki/Beatmap_ranking_procedure/fr.md:19]: please replace `beatmapset` with an equivalent of `beatmap` in French as I did recently

File: wiki/Beatmap_ranking_procedure/Ranking_queue/fr.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  - qualified queue
+  - ranked queue
+  - ranking ETA
+  - file d'attente qualifiée
+  - file d'attente classée
+---
+
+# File d'attente de classement des beatmaps
+
+La **file de classement** gère le passage des [beatmaps](/wiki/Beatmap) de [qualifié](/wiki/Beatmap/Category#qualified) à [classé](/wiki/Beatmap/Category#ranked). Chaque jour, 10 beatmaps de chaque [mode de jeu](/wiki/Game_mode) peuvent passer de la catégorie qualifiée à la catégorie classée, à condition d'avoir été dans la catégorie qualifiée pendant au moins 7 jours. Le moment de la journée où les beatmaps sont classés est aléatoire.
```
Comment (Ganondorfosu) [inline wiki/Beatmap_ranking_procedure/Ranking_queue/fr.md:12]:
```suggestion
La **file de classement** gère le passage des [beatmaps](/wiki/Beatmap) de [qualifié](/wiki/Beatmap/Category#qualified) à [classé](/wiki/Beatmap/Category#ranked). Chaque jour, 10 beatmaps de chaque [mode de jeu](/wiki/Game_mode) peuvent passer de la catégorie qualifiée à la catégorie classée, à condition d'avoir été dans la catégorie qualifiée pendant au moins 7 jours. Le moment de la journée où les beatmaps sont classées est aléatoire.
```

File: wiki/Beatmap_ranking_procedure/Ranking_queue/fr.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+## Disqualification et requalification
+
+Lorsqu'un beatmap est [disqualifié](/wiki/Beatmap_ranking_procedure#nomination-resets), le temps qu'il a passé dans la catégorie qualifié est sauvegardé. Si elle est requalifiée, elle entrera dans la file d'attente comme si elle avait déjà été dans la file d'attente pendant cette durée. Cette possibilité de "sauter" le temps passé dans la file d'attente est plafonnée à 6 jours afin de garantir que les beatmaps restent toujours dans la file d'attente qualifiée pendant au moins une journée entière.
```
Comment (Ganondorfosu) [inline wiki/Beatmap_ranking_procedure/Ranking_queue/fr.md:16]:
```suggestion
Lorsqu'une beatmap est [disqualifiée](/wiki/Beatmap_ranking_procedure#nomination-resets), le temps qu'il a passé dans la catégorie qualifiée est sauvegardé. Si elle est requalifiée, elle entrera dans la file d'attente comme si elle avait déjà été dans la file d'attente pendant cette durée. Cette possibilité de "sauter" le temps passé dans la file d'attente est plafonnée à 6 jours afin de garantir que les beatmaps restent toujours dans la file d'attente qualifiée pendant au moins une journée entière.
```

File: wiki/Beatmap_ranking_procedure/fr.md [hunk 4]
```diff
@@ -1,52 +1,43 @@
...
-outdated: true
-outdated_since: 6e9200e42c1c320aa83b8ac62a2dd9b678bb3eb4
----
+# Procédure de classement des beatmap
 
-# Procédure de classement Beatmap
+*Voir aussi : [Rank (désambiguïsation)](/wiki/Disambiguation/Rank)*
 
-Les [beatmaps](/wiki/Beatmap) peuvent être promu auprès de la communauté en suivant la procédure de classement de la beatmap et en entrant dans la catégorie [Ranked](/wiki/Beatmap/Category#ranked).
+Les [beatmaps](/wiki/Beatmap) peuvent être promus dans la communauté en suivant la procédure de classement des beatmaps et en entrant dans la catégorie [classé](/wiki/Beatmap/Category#ranked).
```
Comment (Ganondorfosu) [inline wiki/Beatmap_ranking_procedure/fr.md:5]:
```suggestion
Les [beatmaps](/wiki/Beatmap) peuvent être promues dans la communauté en suivant la procédure de classement des beatmaps et en entrant dans la catégorie [classée](/wiki/Beatmap/Category#ranked).
```

File: wiki/Beatmap_ranking_procedure/fr.md [hunk 5]
```diff
@@ -1,52 +1,43 @@
...
 
 ## Commentaires
 
-Le créateur d'un beatmap peut marquer ses soumissions comme `Work in Progress` ou `Pending`. Les Beatmaps dans ces deux catégories sont destinés à recevoir les commentaires de la communauté osu!
+Le créateur d'un beatmap peut marquer ses soumissions comme `Work in Progress` ou `En attente`. Les beatmaps de ces deux catégories sont destinés à recevoir les commentaires de la communauté d'osu!
```
Comment (Ganondorfosu) [inline wiki/Beatmap_ranking_procedure/fr.md:9]:
```suggestion
Le créateur d'une beatmap peut marquer ses soumissions comme `Work in Progress` ou `En attente`. Les beatmaps de ces deux catégories sont destinées à recevoir les commentaires de la communauté d'osu!
```

File: wiki/Beatmap_ranking_procedure/fr.md [hunk 6]
```diff
@@ -1,52 +1,43 @@
...
 
-Le [modding](/wiki/Modding) est le processus qui consiste à recevoir des critiques constructives sur une map de pointage et qui est bénéfique à l'amélioration de sa qualité. Les utilisateurs affichent habituellement leurs commentaires sur la page de discussion d'une map ou discutent directement des problèmes avec le créateur de la map.
+Le [modding](/wiki/Modding) est le processus qui consiste à recevoir des critiques constructives sur un beatmap et qui permet d'en améliorer la qualité. Les utilisateurs publient généralement leurs commentaires sur la page de discussion d'une beatmap ou discutent directement avec le créateur de la beatmap.
```
Comment (Ganondorfosu) [inline wiki/Beatmap_ranking_procedure/fr.md:11]:
```suggestion
Le [modding](/wiki/Modding) est le processus qui consiste à recevoir des critiques constructives sur une beatmap et qui permet d'en améliorer la qualité. Les utilisateurs publient généralement leurs commentaires sur la page de discussion d'une beatmap ou discutent directement avec le créateur de la beatmap.
```

File: wiki/Beatmap_ranking_procedure/fr.md [hunk 7]
```diff
@@ -1,52 +1,43 @@
...
 
-Les beatmappers les plus récents ont généralement besoin de modifications importantes pour créer des beatmaps qui répondent aux normes de qualité de la catégorie Ranked, et il leur est souvent demandé de recréer leurs beatmaps à partir de zéro.
+Les nouveaux beatmappers ont généralement besoin d'un important travail de modding pour créer des beatmaps qui répondent aux normes de qualité de la catégorie classée, et il leur est souvent demandé de recréer leurs beatmaps à partir de zéro.
 
-Avant d'être nominés pour le classement, les beatmaps doivent recevoir [hype](/wiki/Glossary#hype) des messages d'au moins cinq utilisateurs différents.
+Avant d'être proposés au classement, les beatmaps doivent recevoir des messages de [hype](/wiki/Glossary#hype) d'au moins cinq utilisateurs différents.
```
Comment (Ganondorfosu) [inline wiki/Beatmap_ranking_procedure/fr.md:15]:
```suggestion
Avant d'être proposées au classement, les beatmaps doivent recevoir des messages de [hype](/wiki/Glossary#hype) d'au moins cinq utilisateurs différents.
```

File: wiki/Beatmap_ranking_procedure/fr.md [hunk 8]
```diff
@@ -1,52 +1,43 @@
...
 
 ## Nominations
 
-Les membres des [Beatmap Nominators](/wiki/People/The_Team/Beatmap_Nominators) (*BNs*) sont des modders expérimentés qui sont capables de nommer des beatmaps pour le classement. Leur rôle est de s'assurer qu'une beatmap répond aux critères du [Ranking Criteria](/wiki/Ranking_Criteria) ainsi que leurs propres normes de qualité personnelles.
+Les membres du groupe des [Beatmap Nominators](/wiki/People/The_Team/Beatmap_Nominators) (*BNs*) sont des moddeurs expérimentés capables de proposer des beatmaps classable. Leur rôle est de s'assurer qu'une beatmap répond aux [critères de classement](/wiki/Ranking_Criteria) ainsi qu'à leurs propres critères de qualité.
```
Comment (Ganondorfosu) [inline wiki/Beatmap_ranking_procedure/fr.md:19]:
```suggestion
Les membres du groupe des [Beatmap Nominators](/wiki/People/The_Team/Beatmap_Nominators) (*BNs*) sont des moddeurs expérimentés capables de proposer des beatmaps classables. Leur rôle est de s'assurer qu'une beatmap répond aux [critères de classement](/wiki/Ranking_Criteria) ainsi qu'à leurs propres critères de qualité.
```

File: wiki/Beatmap_ranking_procedure/fr.md [hunk 9]
```diff
@@ -1,52 +1,43 @@
...
 
-Les membres de la [Nomination Assessment Team](/wiki/People/The_Team/Nomination_Assessment_Team) (*NAT*) peuvent également nommer des maps, mais ce n'est pas leur rôle principal.
+Les membres de la [Nomination Assessment Team](/wiki/People/The_Team/Nomination_Assessment_Team) (*NAT*) peuvent également nommer des beatmaps, mais ce n'est pas leur rôle principal.
```
Comment (Ganondorfosu) [inline wiki/Beatmap_ranking_procedure/fr.md:21]:
```suggestion
Les membres de la [Nomination Assessment Team](/wiki/People/The_Team/Nomination_Assessment_Team) (*NAT*) peuvent également nominer des beatmaps, mais ce n'est pas leur rôle principal.
```

File: wiki/Beatmap_ranking_procedure/fr.md [hunk 10]
```diff
@@ -1,52 +1,43 @@
...
 
-Il est recommandé de recevoir les mods avant de demander des nominations aux BN, cependant cinq hypes sont le seul pré-requis.
+Il est recommandé de recevoir des mods avant de demander des nominations aux BN, mais cinq hypes sont le seul pré-requis.
 
 ## Qualification
 
-Les Beatmaps qui reçoivent deux nominations sont déplacés vers la catégorie [Qualified](/wiki/Beatmap/Category#qualified) de beatmap pendant au moins 7 jours. Les beatmaps qualifiés ont des tableaux de scores, mais ne récompensent pas les joueurs avec des [performance points](/wiki/Performance_Points).
+Les beatmaps qui reçoivent deux nominations pour chaque mode de jeu disponible dans la beatmap sont déplacés dans la catégorie des beatmaps [qualifiés](/wiki/Beatmap/Category#qualified). Les beatmaps qualifiés disposent d'un classement des scores, mais ne récompensent pas les joueurs avec des [points de performance](/wiki/Performance_Points). Lorsqu'une beatmap reste dans la catégorie qualifié pendant au moins 7 jours, la [file d'attente de classement](Ranking_queue) peut le faire passer dans la catégorie [classée](#classée).
```
Comment (Ganondorfosu) [inline wiki/Beatmap_ranking_procedure/fr.md:27]:
```suggestion
Les beatmaps qui reçoivent deux nominations pour chaque mode de jeu disponible dans la beatmap sont déplacées dans la catégorie des beatmaps [qualifiées](/wiki/Beatmap/Category#qualified). Les beatmaps qualifiées disposent d'un classement des scores, mais ne récompensent pas les joueurs avec des [points de performance](/wiki/Performance_Points). Lorsqu'une beatmap reste dans la catégorie qualifiée pendant au moins 7 jours, la [file d'attente de classement](Ranking_queue) peut la faire passer dans la catégorie [classée](#classée).
```

File: wiki/Beatmap_ranking_procedure/fr.md [hunk 11]
```diff
@@ -1,52 +1,43 @@
...
 
-Cette catégorie de beatmaps existe pour fournir une plus grande exposition à la communauté osu! avec l'espoir de découvrir des problèmes potentiels. Les problèmes peuvent être signalés aux membres du BN et du NAT à partir de la page de discussion d'une beatmap. Contrairement aux beatmaps en attente, les beatmaps qualifiés ne peuvent pas être mis à jour par leurs créateurs, donc les commentaires ne peuvent être appliqués qu'après avoir demandé un [reset nomination](#nomination-resets).
+Cette catégorie de beatmap existe pour offrir aux beatmaps une plus grande exposition à la communauté d'osu! dans l'espoir de découvrir des problèmes potentiels. Les problèmes peuvent être signalés aux membres du BN et du NAT à partir de la page de discussion d'une beatmap. Contrairement aux beatmaps en attente, les beatmaps qualifiés ne peuvent pas être mis à jour par leurs créateurs. Les commentaires ne peuvent donc être appliqués qu'après avoir demandé une [réinitialisation de la nomination](#réinitialisation-de-la-nomination).
```
Comment (Ganondorfosu) [inline wiki/Beatmap_ranking_procedure/fr.md:29]:
```suggestion
Cette catégorie de beatmap existe pour offrir aux beatmaps une plus grande exposition à la communauté d'osu! dans l'espoir de découvrir des problèmes potentiels. Les problèmes peuvent être signalés aux BN et aux membres de la NAT à partir de la page de discussion d'une beatmap. Contrairement aux beatmaps en attente, les beatmaps qualifiées ne peuvent pas être mises à jour par leurs créateurs. Les commentaires ne peuvent donc être appliqués qu'après avoir demandé une [réinitialisation de la nomination](#réinitialisation-de-la-nomination).
```

File: wiki/Beatmap_ranking_procedure/fr.md [hunk 12]
```diff
@@ -1,52 +1,43 @@
...
 
-### Déménagement au classement
+## Réinitialisation de la nomination
 
-Les beatmaps qualifiés sont placés dans une file d'attente pour le classement. 8 beatmaps de chaque mode de jeu peuvent passer de Qualifié à Classé, à condition qu'ils aient été dans la file d'attente pendant au moins 7 jours. Cela signifie que certaines beatmaps restent qualifiés pendant un peu plus de 7 jours.
+Les réinitialisations des nominations permettent de s'assurer que les modérateurs, les BN et les membres du NAT vérifient la version la plus récente d'une beatmap avant qu'il n'entre dans la file d'attente de classement.
```
Comment (Ganondorfosu) [inline wiki/Beatmap_ranking_procedure/fr.md:33]:
```suggestion
Les réinitialisations des nominations permettent de s'assurer que les modérateurs, les BN et les membres de la NAT vérifient la version la plus récente d'une beatmap avant qu'elle n'entre dans la file d'attente de classement.
```

File: wiki/Beatmap_ranking_procedure/fr.md [hunk 13]
```diff
@@ -1,52 +1,43 @@
...
 
-Lorsque une beatmap est disqualifié, sa position dans la file d'attente de classement est sauvegardée. S'il est re-qualifié, il sera placé à sa position précédente dans la file d'attente de classement. Par exemple, si une beatmap est qualifié pendant 2 jours, puis disqualifié et re-qualifié, il restera dans la catégorie Qualifié pendant environ 5 jours de plus. Si la position sauvegardée dans la file d'attente est à moins de 24 heures du classement, elle sera remise à 24 heures lors de l'entrée dans la catégorie Qualifié.
+Les nominations peuvent être réinitialisé lorsque le créateur met à jour sa beatmap ou lorsqu'un membre du BN ou du NAT trouve un problème dans une difficulté nommé. Elles peuvent également être réinitialisé par les membres de la [Global Moderation Team](/wiki/People/The_Team/Global_Moderation_Team) lorsque la raison est un contenu inapproprié. Si une difficulté est qualifié lorsque ses nominations sont réinitialisé, il sera disqualifié, ce qui le supprimera de la file d'attente de classement.
```
Comment (Ganondorfosu) [inline wiki/Beatmap_ranking_procedure/fr.md:35]:
```suggestion
Les nominations peuvent être réinitialisées lorsque le créateur met à jour sa beatmap ou lorsqu'un BN ou un membre de la NAT trouve un problème dans une difficulté nommée. Elles peuvent également être réinitialisées par les membres de la [Global Moderation Team](/wiki/People/The_Team/Global_Moderation_Team) lorsque la raison est un contenu inapproprié. Si une difficulté est qualifiée lorsque ses nominations sont réinitialisées, elle sera disqualifiée, ce qui la supprimera de la file d'attente de classement.
```

File: wiki/Beatmap_ranking_procedure/fr.md [hunk 14]
```diff
@@ -1,52 +1,43 @@
...
 
-## Réinitialisation des nominations
+Les membres du BN et du NAT opposent parfois leur [veto](/wiki/People/The_Team/Beatmap_Nominators/Beatmap_Veto) aux difficultés qui, selon eux, présentent des problèmes de qualité sans rapport avec les critères de classement.
```
Comment (Ganondorfosu) [inline wiki/Beatmap_ranking_procedure/fr.md:37]:
```suggestion
Les BN et les membres de la NAT opposent parfois leur [veto](/wiki/People/The_Team/Beatmap_Nominators/Beatmap_Veto) aux difficultés qui, selon eux, présentent des problèmes de qualité sans rapport avec les critères de classement.
```

File: wiki/Beatmap_ranking_procedure/fr.md [hunk 15]
```diff
@@ -1,52 +1,43 @@
...
 
-Les réinitialisations de nomination aident à s'assurer que les modders, les BNs et les membres du NAT vérifient la version la plus récente d'une beatmap avant qu'il n'entre dans la file d'attente de classement.
+## Classée
 
-Les nominations peuvent être réinitialisées quand le créateur d'une beatmap met à jour l'ensemble ou quand un membre du BN ou du NAT trouve un problème dans une beatmap nommé. Elles peuvent aussi être réinitialisées par les membres de la [Global Moderation Team](/wiki/People/The_Team/Global_Moderation_Team) quand la raison implique un contenu inapproprié. Si une beatmap est qualifié lorsque ses nominations sont réinitialisées, l'ensemble sera disqualifié, le retirant de la file de classement.
+Les beatmaps qui ont franchi l'étape de qualification ont terminé la procédure de classement. Ils disposent de tableaux de classement des scores et récompensent les joueurs par des points de performance.
```
Comment (Ganondorfosu) [inline wiki/Beatmap_ranking_procedure/fr.md:41]:
```suggestion
Les beatmaps qui ont franchi l'étape de qualification ont terminé la procédure de classement. Elles disposent de tableaux de classement des scores et récompensent les joueurs par des points de performance.
```

File: wiki/Beatmap_ranking_procedure/fr.md [hunk 16]
```diff
@@ -1,52 +1,43 @@
...
 
-Membres du BN et du NAT à l'occasion [veto](/wiki/People/The_Team/Beatmap_Nominators/Beatmap_Veto) les ensembles de battants qu'ils estiment avoir des problèmes de qualité qui ne sont pas liés aux critères de classement.
-
-## Classé
-
-Les beatmaps qui sont passés par l'étape de qualification ont terminé la procédure de classement. Ils ont des tableaux de classement et récompensent les joueurs avec des points de performance.
-
-Les beatmaps classées ne sont plus classées que dans des circonstances exceptionnelles lorsque des problèmes sont découverts peu après qu'elles aient atteint le statut de Ranked.
+Les beatmaps classés ne sont déclassés que dans des circonstances exceptionnelles, lorsque des problèmes sont découverts peu après qu'ils aient atteint le statut classé.
```
Comment (Ganondorfosu) [inline wiki/Beatmap_ranking_procedure/fr.md:43]:
```suggestion
Les beatmaps classées ne sont déclassées que dans des circonstances exceptionnelles, lorsque des problèmes sont découverts peu après qu'ils aient atteint le statut classé.
```

**Review by Ganondorfosu — CHANGES_REQUESTED** [review]

File: wiki/Beatmap_ranking_procedure/fr.md [hunk 17]
```diff
@@ -1,52 +1,43 @@
...
+Le [modding](/wiki/Modding) est le processus qui consiste à recevoir des critiques constructives sur une beatmap et qui permet d'en améliorer la qualité. Les utilisateurs publient généralement leurs commentaires sur la page de discussion d'une beatmap ou discutent directement avec le créateur de la beatmap.
 
-Les beatmappers les plus récents ont généralement besoin de modifications importantes pour créer des beatmaps qui répondent aux normes de qualité de la catégorie Ranked, et il leur est souvent demandé de recréer leurs beatmaps à partir de zéro.
+Les nouveaux beatmappers ont généralement besoin d'un important travail de modding pour créer des beatmaps qui répondent aux normes de qualité de la catégorie classée, et il leur est souvent demandé de recréer leurs beatmaps à partir de zéro.
 
-Avant d'être nominés pour le classement, les beatmaps doivent recevoir [hype](/wiki/Glossary#hype) des messages d'au moins cinq utilisateurs différents.
+Avant d'être proposées au classement, les beatmaps doivent recevoir des messages de [hype](/wiki/Glossary#hype) d'au moins cinq utilisateurs différents.
```
Comment (DoryanR) [inline wiki/Beatmap_ranking_procedure/fr.md:15]:
```suggestion
Avant d'être proposées au classement, les beatmaps doivent recevoir des messages de [hype](/wiki/Beatmap/Hype) d'au moins cinq utilisateurs différents.
```

**Comment by TicClick** [comment]
if @Ganondorfosu approves, this can be merged

File: wiki/Beatmap_ranking_procedure/Ranking_queue/fr.md [hunk 18]
```diff
@@ -0,0 +1,16 @@
...
+  - qualified queue
+  - ranked queue
+  - ranking ETA
+  - file d'attente qualifiée
+  - file d'attente classée
+---
+
+# File d'attente de classement des beatmaps
+
+La **file de classement** gère le passage des [beatmaps](/wiki/Beatmap) de [qualifié](/wiki/Beatmap/Category#qualified) à [classé](/wiki/Beatmap/Category#ranked). Chaque jour, 10 beatmaps de chaque [mode de jeu](/wiki/Game_mode) peuvent passer de la catégorie qualifiée à la catégorie classée, à condition d'avoir été dans la catégorie qualifiée pendant au moins 7 jours. Le moment de la journée où les beatmaps sont classées est aléatoire.
```
Comment (Ganondorfosu) [inline wiki/Beatmap_ranking_procedure/Ranking_queue/fr.md:12]:
```suggestion
La **file de classement** gère le passage des [beatmaps](/wiki/Beatmap) de la catégorie [qualifiée](/wiki/Beatmap/Category#qualified) à la catégorie [classée](/wiki/Beatmap/Category#ranked). Chaque jour, 10 beatmaps de chaque [mode de jeu](/wiki/Game_mode) peuvent passer de la catégorie qualifiée à la catégorie classée, à condition d'avoir été dans la catégorie qualifiée pendant au moins 7 jours. Le moment de la journée où les beatmaps sont classées est aléatoire.
```

File: wiki/Beatmap_ranking_procedure/Ranking_queue/fr.md [hunk 19]
```diff
@@ -0,0 +1,16 @@
...
+
+## Disqualification et requalification
+
+Lorsqu'une beatmap est [disqualifiée](/wiki/Beatmap_ranking_procedure#nomination-resets), le temps qu'il a passé dans la catégorie qualifiée est sauvegardé. Si elle est requalifiée, elle entrera dans la file d'attente comme si elle avait déjà été dans la file d'attente pendant cette durée. Cette possibilité de "sauter" le temps passé dans la file d'attente est plafonnée à 6 jours afin de garantir que les beatmaps restent toujours dans la file d'attente qualifiée pendant au moins une journée entière.
```
Comment (Ganondorfosu) [inline wiki/Beatmap_ranking_procedure/Ranking_queue/fr.md:16]:
```suggestion
Lorsqu'une beatmap est [disqualifiée](/wiki/Beatmap_ranking_procedure#nomination-resets), le temps qu'elle a passé dans la catégorie qualifiée est sauvegardé. Si elle est requalifiée, elle entrera dans la file d'attente comme si elle avait déjà été dans la file d'attente pendant cette durée. Cette possibilité de "sauter" le temps passé dans la file d'attente est plafonnée à 6 jours afin de garantir que les beatmaps restent toujours dans la file d'attente qualifiée pendant au moins une journée entière.
```

File: wiki/Beatmap_ranking_procedure/fr.md [hunk 20]
```diff
@@ -1,52 +1,43 @@
...
+Les beatmaps qui ont franchi l'étape de qualification ont terminé la procédure de classement. Elles disposent de tableaux de classement des scores et récompensent les joueurs par des points de performance.
 
-Membres du BN et du NAT à l'occasion [veto](/wiki/People/The_Team/Beatmap_Nominators/Beatmap_Veto) les ensembles de battants qu'ils estiment avoir des problèmes de qualité qui ne sont pas liés aux critères de classement.
-
-## Classé
-
-Les beatmaps qui sont passés par l'étape de qualification ont terminé la procédure de classement. Ils ont des tableaux de classement et récompensent les joueurs avec des points de performance.
-
-Les beatmaps classées ne sont plus classées que dans des circonstances exceptionnelles lorsque des problèmes sont découverts peu après qu'elles aient atteint le statut de Ranked.
+Les beatmaps classées ne sont déclassées que dans des circonstances exceptionnelles, lorsque des problèmes sont découverts peu après qu'ils aient atteint le statut classé.
```
Comment (Ganondorfosu) [inline wiki/Beatmap_ranking_procedure/fr.md:43]:
```suggestion
Les beatmaps classées ne sont déclassées que dans des circonstances exceptionnelles, lorsque des problèmes sont découverts peu après qu'elles aient atteint le statut classé.
```

**Review by Ganondorfosu — CHANGES_REQUESTED** [review]

**Review by Ganondorfosu — APPROVED** [review]

**Review by TicClick — APPROVED** [review]

**Pull request merged**